FogCreek/Glitch-Community
# Happy-seaplane: add a timeout on calls to friendly-words

## Links
* https://happy-seaplane.glitch.me/
* https://app.clubhouse.io/glitch/story/1601/investigate-what-happens-when-friendly-words-fails-to-respond-in-create-team-pop

## GIF/Screenshots:
what will happen when the request times out/fails:
![image](https://user-images.githubusercontent.com/4480480/68978614-e37da900-07c0-11ea-9dba-d4847c5c9f39.png)

## Changes:
* add a timeout to the call to friendly-words

## How To Test:
* Change the url in models/words to a remix of friendly-words that is set up to fail requests, like polite-macaroon.glitch.me (only fails on the endpoint used by the create team pop). See that the fallback is displayed.
